src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -1,38 +1,31 @@
-// Copyright (c) FIRST and other WPILib contributors.
-// Open Source Software; you can modify and/or share it under the terms of
-// the WPILib BSD license file in the root directory of this project.
-
 package frc.robot.subsystems;
 
-import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
 import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj.drive.DifferentialDrive;
 import edu.wpi.first.wpilibj.motorcontrol.MotorController;
 import edu.wpi.first.wpilibj.motorcontrol.MotorControllerGroup;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.DriveTrainContants;
 
-public class DriveTrain extends SubsystemBase {
-  /** Creates a new DriveBase. */
-  public DriveTrain() {
-    MotorController m_frontLeft = new WPI_VictorSPX(7);
-    MotorController m_backLeft = new WPI_VictorSPX(6);
-    MotorController m_frontRight = new WPI_VictorSPX(8);
-    MotorController m_backRight = new WPI_VictorSPX(5);
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
 
-    MotorControllerGroup m_left = new MotorControllerGroup(m_frontLeft, m_backLeft);
-    MotorControllerGroup m_right = new MotorControllerGroup(m_frontRight, m_backRight);
+public class DriveTrain extends SubsystemBase{
+    private MotorController m_frontLeft = new WPI_VictorSPX(DriveTrainContants.frontLeftMotorId);
+    private MotorController m_backLeft = new WPI_VictorSPX(DriveTrainContants.backLeftMotorId);
+    private MotorController m_frontRight = new WPI_VictorSPX(DriveTrainContants.frontRightMotorId);
+    private MotorController m_backRight = new WPI_VictorSPX(DriveTrainContants.backRightMotorId);
+    
+    private MotorControllerGroup m_leftGroup = new MotorControllerGroup(m_frontLeft, m_backLeft );
+    private MotorControllerGroup m_rightGroup = new MotorControllerGroup(m_frontRight, m_backRight);
 
-    DifferentialDrive m_robotDrive = new DifferentialDrive(m_left, m_right);
+    private DifferentialDrive driveSystem = new DifferentialDrive(m_leftGroup, m_rightGroup);
```

Comment:
Please prefix class member variables with `m_`
```suggestion
    private DifferentialDrive m_driveSystem = new DifferentialDrive(m_leftGroup, m_rightGroup);
```